scripts/mysql-user.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 MYSQL_OPTIONS="-h $MYSQL_HOST --user=$MYSQL_MASTER_USER --password=$MYSQL_MASTER_PASSWORD"
 MYSQL_USER=$(aws secretsmanager get-secret-value --secret-id ${MYSQL_SECRET_ARN}|jq .SecretString -r|jq .username -r)
-MYSQL_PASSWORD=$(aws secretsmanager get-secret-value --secret-id ${MYSQL_SECRET_ARN}|jq .SecretString -r|jq .password -r)
+MYSQL_PASSWORD=$(aws secretsmanager get-secret-value --secret-id ${MYSQL_SECRET_ARN} --region ${AWS_REGION}|jq .SecretString -r|jq .password -r)
```

Comment:
I realize I’m late to the game, but don’t you need to add the `—region` flag to the `aws secretsmanager` command that is getting the `MYSQL_USER` value too?

Reply:
ahh..nice catch.
somehow it worked as expected but will raise another PR to make it consistent.

Reply:
@barnharts4  - new PR https://github.com/ExpediaGroup/apiary-data-lake/pull/110